lib-dmtf/model/Links.go

```diff
@@ -48,5 +48,4 @@ type Link struct {
 }
 
 // Oem holds the vendor specific details which is addtional to redfish contents
-type Oem struct {
-}
+type Oem interface{}
```

Comment:
Is there any chance of exception if we use interface ? Have we tested this thoroughly ?

Reply:
@Bharath-KKB yes I have tested it. It is working fine with the current implementations